src/drivers/actuators/voxl_esc/voxl_esc.cpp

```diff
@@ -1038,6 +1063,12 @@ bool VoxlEsc::updateOutputs(bool stop_motors, uint16_t outputs[MAX_ACTUATORS],
 			_esc_chans[i].rate_req = 0;
 
 		} else {
+			if (_extended_rpm) {
+				if (outputs[i] > 65530) outputs[i] = 65530;
```

Comment:
Perhaps a constant definition of 65530 and 32766 with some comments about why those values were chosen

Reply:
I will make this happen